src/main/java/com/raisetech/crudsample/service/UserServiceImpl.java

```diff
@@ -29,17 +30,17 @@ public Optional<User> findById(int id) {
 
     @Override
     public void insertUser(InsertForm insertForm) {
+        System.out.println(insertForm.getId());
         userMapper.insertUser(insertForm);
```

Comment:
下記のように採番されたIDまたはInsertFormを返してControllerではそれを使うようにしたいです。
```suggestion
    public int insertUser(InsertForm insertForm) {
        userMapper.insertUser(insertForm);
        return insertForm.getId();
```

```suggestion
    public InsertForm insertUser(InsertForm insertForm) {
        userMapper.insertUser(insertForm);
        return insertForm;
```
（ただし、私個人の考え方としてFormはControllerの持ち物としてServiceには持ち込ませないような実装にしたいです。これはServiceとControllerの責任分界点を明確にしたいからです。）

Reply:
ありがとうございます。自分では書けないけれど、読むと納得です。
戻り値にintをとるような形で実装してみましたが、どうでしょうか。